src/global_distrib/mod_global_distrib_outgoing_conns_sup.erl

```diff
@@ -45,9 +42,23 @@ get_connection(Server) ->
 
 init(_) ->
     SupFlags = #{ strategy => one_for_one, intensity => 5, period => 5 },
-    {ok, {SupFlags, []}}.
+    RefresherSpec = #{
+      id => mod_global_distrib_hosts_refresher,
+      start => {mod_global_distrib_hosts_refresher, start_link, []},
+      restart => transient,
+      shutdown => 5000
+    },
+    {ok, {SupFlags, [RefresherSpec]}}.
```

Comment:
Since `mod_global_distrib_hosts_refresher` is a `gen_mod`-type module, its process should probably be started in its `start/2` callback instead of being hardcoded here.